server/app/services/ambassadors.js

```diff
@@ -93,9 +93,8 @@ async function signup(json, verification, carrierLookup) {
   let existing_ambassador = null
 
   if (alloy_response) {
-    existing_ambassador = await neode.first("Ambassador", {
-      alloy_person_id: alloy_response.data.alloy_person_id,
-    })
+    const alloy_person_id=alloy_response.data.alloy_person_id.toString()
```

Comment:
Don't call `toString()` — it'll crash if called on `null`.  If you want to convert to string, safer to say

    '' + alloy_response.data.alloy_person_id

Reply:
will do